guide/miscellaneous/making-an-eval-command.md

```diff
@@ -0,0 +1,110 @@
+# Making an Eval command
+
+## Introduction
+
+### What is eval?
+
+The eval() function evaluates JavaScript code represented as a string.
+For instance, `eval(1 + 1)` will return `2` and `eval(client.user.tag)` will return the bot's tag, e.g. `MyBot#0001`.
+
+### Why eval can be dangerous?
+
+eval() is a dangerous function, which executes the passed code, with the privileges of the caller. If you run eval() with a string that could be affected by a malicious party, you may end up running malicious code on your machine. More importantly, a third-party code can see the scope in which eval() was invoked, which can lead to possible attacks in ways to which the similar Function is not susceptible.
+
+In other words, eval is dangerous and if somehow someone else has access to it, the person can run (malicious) code with full access.
+Furthermore, anyone with the access to it can get the app's secrets and credentials - such as bot token, API keys, so on.
+
+:::warning
+Note: eval() is also slower than the alternatives, since it has to invoke the JS interpreter, while many other constructs are optimized by modern JS engines.
+:::
+
+## Implementation
+
+### Securing eval
+
+Basically, we will be limiting the eval command only to the owner.
+To do that, we will add an condition, which will return if the author's ID is not equals to owner's ID.
+
+<!-- eslint-skip -->
+
+```js
+if (message.author.id !== 'ownerID') return;
+```
+
+Additionally, you can add your ID in the config file:
+
+```json
+{
+    "prefix": "!",
+    "token": "your-token-goes-here",
+    "owner": "Owner-ID-here"
+}
+```
+
+<!-- eslint-skip -->
+
+```js
+const { owner } = require('../config.json');
+if (message.author.id !== owner) return;
+```
+
+### Cleaning eval
+
+We'll be creating a function that prevents the use of actual mentions within the return line by adding a zero-width character between the @ and the first character of the mention - blocking the mention from happening.
+
+<!-- eslint-skip -->
+
+```js
+const clean = content => {
+    if (typeof content === 'string') {
+        return content.replace(/`/g, '`' + String.fromCharCode(8203)).replace(/@/g, '@' + String.fromCharCode(8203));
+    } else {
+        return content;
+    }
+}
+```
+
+### Object to String representation
+
+We can convert object to string representation by using an in-built Node.js module - `util`. Specifically, `util#inspect()`
+So what is `util#inspect`? `util.inspect()` method returns a string representation of object that is intended for debugging.
```

Comment:
Nitpick: 3 different ways to mention `Util#inspect` :P

Reply:
Will replace in new commit and thanks.